lib/joken_jwks/default_strategy_template.ex

```diff
@@ -219,12 +223,18 @@ defmodule JokenJwks.DefaultStrategyTemplate do
         case EtsCache.check_state() do
           # no need to re-fetch
           0 ->
-            JokenJwks.log(:debug, opts[:log_level], "Re-fetching cache is not needed.")
+            :telemetry.execute(~w/joken_jwks ets_cache not_needed/a, %{}, %{
```

Comment:
For all telemetry events it is important to have the name of the module or something that would identify the specific handler. This library is meant to be used even with several instances on the same app.

Reply:
Ok. I added an extra `:module` key on `metadata` to register the module's name on both modules that are making telemetry calls.